desktop/libs/Scripts/sync-service/index.js

```diff
@@ -0,0 +1,118 @@
+var server = require('http').createServer()
+  , url = require('url')
+  , express = require('express')
+  , app = express()
+  , cors = require('cors')
+  , WebSocketServer = require('ws').Server
+  , port = 4080;
+
+var io = require('socket.io')(4082)
+let onImage = () => {}
+let onLayout = () => {}
+let onActiveScene = () => {}
+io.on('connection', (socket) => {
+  onImage = (image, sceneName) => {
+    socket.emit('image', { sceneName, image })
+  }
+  onLayout = (layout, sceneName) => {
+    socket.emit('layout', {sceneName, layout })
+  }
+  onActiveScene = (sceneName) => {
+    socket.emit('activeScene', {sceneName})
+  }
+})
+
+app.use(cors())
+
+var imageForScene = {}
+var layoutForScene = {}
+var activeScene = null
+
+app.get('/image/:sceneName', function (req, res) {
+  res.writeHead(200, {'Content-Type': 'image/png' });
+  if (!imageForScene[req.params.sceneName]) {
+    res.status(404).end()
+    return
+  }
+  res.end(imageForScene[req.params.sceneName], 'binary')
+});
+
+app.get('/layout/:sceneName', function (req, res) {
+  if (!layoutForScene[req.params.sceneName]) {
+    res.status(404).end()
+    return
+  }
+  res.json(layoutForScene[req.params.sceneName])
+})
+
+// IMAGE DATA
+var wss = new WebSocketServer({ port: 4081 });
+
+function getHeaderInfo(data) {
```

Comment:
Probably should add a comment outlining the format of this stuff, or links pointing to the docs if it's a standard format